products/proxmox-ve.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Proxmox VE
+category: os
+iconSlug: proxmox
+permalink: /proxmox-ve
+alternate_urls:
+-   /pve
+-   /proxmoxve
+activeSupportColumn: Supported
+releasePolicyLink: https://pve.proxmox.com/wiki/Roadmap
```

Comment:
A page on the Proxmox website explaining what's in the product description would be better (if such page exists).

> Proxmox VE versions are supported at least as long as the corresponding Debian Version is oldstable. Proxmox VE uses a rolling release model and using the latest stable version is always recommended.

Reply:
For example this link: https://pve.proxmox.com/wiki/FAQ.